RNSquareInAppPayments.podspec

```diff
@@ -1,7 +1,7 @@
 
 Pod::Spec.new do |s|
   s.name         = "RNSquareInAppPayments"
-  s.version      = "1.5.0"
+  s.version      = "1.4.0"
```

Comment:
We don't change this version, instead, it should be 1.5.1

Reply:
It's done.